src/main/client/type.c

```diff
@@ -747,7 +746,6 @@ static int AerospikeClient_Type_Init(AerospikeClient *self, PyObject *args,
         }
     }
 
-    as_policies_init(&config.policies);
```

Comment:
Why remove this? Doesn't it setup the client's default policies?

Reply:
Calling as_policies_init() here was causing a linker error on Windows ([logs](https://github.com/aerospike/aerospike-client-python/actions/runs/7673357301/job/20915820089#step:5:14928) of Windows build).

```
  type.obj : error LNK2001: unresolved external symbol as_policies_init
```

I think it's because `as_policies_init()` is [marked as private](https://github.com/aerospike/aerospike-client-c/blob/29860294a714d254c71d4325d8242b66f7a773eb/src/include/aerospike/as_policy.h#L1857).

And earlier in the calling function `AerospikeClient_Type_Init()`, it calls `as_config_init()` [here](https://github.com/aerospike/aerospike-client-python/blob/CLIENT-2322-windows-support/src/main/client/type.c#L566) which already calls `as_policies_init()` under the hood for the client config's policies ([source code for as_config_init](https://github.com/aerospike/aerospike-client-c/blob/29860294a714d254c71d4325d8242b66f7a773eb/src/main/aerospike/as_config.c#L54)). So I think this line is safe to remove.